src/App.jsx

```diff
@@ -14,33 +47,78 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reverse, setReverse] = useState('');
+  const visibleGoods = getPrepareGoods(goodsFromServer, { sortField, reverse });
+  const FILED_REVERSE = !reverse;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const fnReset = () => {
+    setSortField('');
+    setReverse('');
+  };
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          onClick={() => setSortField(SORT_FILED_ALPHABET)}
+          type="button"
+          className={cn({
+            'is-light': sortField !== SORT_FILED_ALPHABET,
+            button: true,
+            'is-info': true,
+          })}
+        >
+          Sort alphabetically
+        </button>
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+        <button
+          onClick={() => setSortField(SORT_FILED_LENGT)}
+          type="button"
+          className={cn({
+            'is-light': sortField !== SORT_FILED_LENGT,
+            button: true,
+            'is-success': true,
+          })}
+        >
+          Sort by length
+        </button>
+
+        <button
+          onClick={() => setReverse(FILED_REVERSE)}
```

Comment:
```suggestion
          onClick={() => setReverse(prev => !prev)}
```
Is is safer to pass callback to state updater if new state value based on previous one